proposals/csharp-9.0/function-pointers.md

```diff
@@ -1,4 +1,4 @@
-﻿# Function Pointers
+# Function Pointers
```

Comment:
What changed here? Did the file encoding get set to something different?

Reply:
It was a red dot that showed up in the edit view. You can see it here: https://github.com/dotnet/csharplang/edit/main/proposals/csharp-9.0/function-pointers.md

Reply:
Right, I believe that is the UTF-8 BOM. What I'm trying to determine is whether it should be removed 🙂

Reply:
@gewarren did you determine what this change was?

Reply:
@333fred It was the BOM, as you suspected. (You can [see it in the blue status bar](https://github.dev/dotnet/csharplang/blob/26e70504f3505398fe8b21b85ff5f74ab3f07e7e/proposals/csharp-9.0/function-pointers.md) for the previous version of the file.) I looked at a couple other .md files in this repo and they didn't have the BOM.

Reply:
Thanks.